package.json

```diff
@@ -4,12 +4,15 @@
   "private": true,
   "main": "src/index.js",
   "scripts": {
-    "start": "./start.sh",
-    "dev": "./dev.sh",
-    "build": "NODE_ENV=production webpack . && podman image build -t play-reading .",
+    "start": "./run.sh",
+    "dev": "./run.sh dev",
+    "css": "cleancss -o public/main.css public/css/{normalize.css,milligram.css,style.css}",
+    "build": "NODE_ENV=production webpack . && npm run css",
+    "build-db": "podman image build -t play-reading .",
```

Comment:
In what situation would we need to separately build the database?

Reply:
It's mostly a "what situation would we want to build the database at the same time when we're building the codebase" question.  
If we do `npm run build` then `npm start`, npm start takes care of building the image.  
Running `npm run dev` also builds the image itself.  
When deploying, we build the container on the target system, not locally.  
We may not even need this build-db at all. Thoughts?

Reply:
I feel like we don't need it. As you covered, the times when we are running code that needs it, we are building the container already. If we are in a situation where we need to be messing with the container, I would assume we would be messing around more with direct CLI podman commands rather than npm scripts.